test-data/virtualmachine-missing-attributes.json

```diff
@@ -0,0 +1,127 @@
+                },
+                "spec": {
+                    "sourceRef": {
+                        "kind": "DataSource",
+                        "name": "rhel9",
+                        "namespace": "openshift-virtualization-os-images"
+                    },
+                    "storage": {
+                        "resources": {
+                            "requests": {
+                                "storage": "30Gi"
+                            }
+                        }
+                    }
+                }
+            }
+        ],
+        "template": {
+            "spec": {
+                "domain": {
+                    "devices": {
+                        "disks": [
+                            {
+                                "disk": {
+                                    "bus": "virtio"
+                                },
+                                "name": "rootdisk"
+                            },
+                            {
+                                "disk": {
+                                    "bus": "virtio"
+                                },
+                                "name": "cloudinitdisk"
+                            }
+                        ],
+                        "interfaces": [
+                            {
+                                "macAddress": "00:f0:70:00:00:00",
+                                "masquerade": {},
+                                "model": "virtio",
+                                "name": "default"
+                            }
+                        ],
+                        "networkInterfaceMultiqueue": true,
+                        "rng": {}
+                    },
+                    "features": {
+                        "acpi": {},
+                        "smm": {
+                            "enabled": true
+                        }
+                    },
+                    "firmware": {
+                        "bootloader": {
+                            "efi": {}
+                        }
+                    },
+                    "machine": {
+                        "type": "pc-q35-rhel9.2.0"
+                    },
+                    "resources": {}
+                },
+                "networks": [
+                    {
+                        "name": "default",
+                        "pod": {}
+                    }
+                ],
+                "terminationGracePeriodSeconds": 180,
+                "volumes": [
+                    {
+                        "cloudInitConfigDrive": {
+                            "userData": "#cloud-config\nuser: clclc-user"
+                        },
+                        "name": "cloudinitdisk"
+                    },
+                    {
+                        "name": "cloudinitdisk",
+                        "cloudInitNoDisk": {
+                            "userData": "#cloud-config\nchpasswd:\nexpire: false\npassword: q9hy-mzra-3vk9\nuser: rhel"
```

Comment:
Remove this line or change password to make it clear it's a dummy password and not a credential leak.